LayoutTests/imported/w3c/web-platform-tests/pointerlock/pointerlock_promise.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<body>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+</head>
+<body>
+    <h2>Description</h2>
+    <p>This test validates that pointer lock returns a Promise.</p>
+    <hr/>
+
+    <button id="Button">lockTarget</button>
+    
+    <div id="target">Target</div>
+
+    <script type="text/javascript" >
+        let button = document.getElementById('Button');
+        let target = document.getElementById('target');
```

Comment:
Please only use `let` if the variable is going to change, otherwise please use `const`...

```suggestion
        const button = document.getElementById('Button');
        const target = document.getElementById('target');
```

Reply:
Same with the ones below...

Reply:
Done.